tests/beta/test_tensornet.py

```diff
@@ -19,8 +19,39 @@
 import math
 
 import pytest
+
+# TODO: remove the following skip when Tensornet has been ported to
+# Qubit device, and the gate imports above are removed.
+tensorflow = pytest.importorskip("tensorflow", minversion="2.0")
```

Comment:
This is because the new import statement below will only work if TensorFlow 2.0 is installed.